enigma/build.gradle

```diff
@@ -11,7 +11,7 @@ dependencies {
     implementation 'net.fabricmc:procyon-fabric-compilertools:0.5.35.13'
     implementation 'net.fabricmc:cfr:0.0.3'
 
-    proGuard 'com.guardsquare:proguard-base:7.1.0-beta1'
+    proGuard 'com.guardsquare:proguard-base:7.1.0-beta3'
```

Comment:
https://github.com/Guardsquare/proguard/commit/036291e8fb91d4aac26fc2ac2dbbe4d09ad5ae32 They are now on beta 5

Reply:
Seems to just be beta 4.